src/distr/distribution.rs

```diff
@@ -110,7 +110,7 @@ pub trait Distribution<T> {
     }
 }
 
-impl<'a, T, D: Distribution<T> + ?Sized> Distribution<T> for &'a D {
+impl<T, D: Distribution<T> + ?Sized> Distribution<T> for &D {
```

Comment:
Clippy is pedantically complaining that we're being too pedantic?

(Sorry, I'll drop the sarcasm! This is fine.)

Reply:
Heh, to be fair, it's a good lint for raising awareness that such elision is allowed. As an "old-timer" who was using Rust since before 1.31, I automatically was using the "pedantic" style everywhere.